.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - REACT_APP_AUTH0_DOMAIN: dev-pinster-illuminusltd.auth0.com
       - REACT_APP_AUTH0_AUDIENCE: https://api-dev.pinster.io
       - REACT_APP_AUTH0_SCOPE: openid profile
-      - REACT_APP_AUTH0_RESPONSE_TYPE: token id_token"
```

Comment:
Why don't you need the id_token? How will we get their name to save as the `display_name`?

Reply:
The auth0 response is including an `idTokenPayload` object even without requesting that. So I sill have `display_name`.

Reply:
rebased to restore this

Reply:
Oh neat, cool beans